### PR TITLE
DOCS-1816-IA-registry-scanner-RN

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CC 3.18

Issue:
https://tigera.atlassian.net/browse/DOCS-1816

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx [hunk 1]
```diff
@@ -10,33 +10,45 @@ description: What's new, and why features provide value for upgrading.
 
 <!-- Updates from CE 3.18 -->
 
-### Security event management
+#### Image Assurance registry scanner
```
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: All these feature titles should be H3

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx [hunk 2]
```diff
@@ -10,33 +10,45 @@ description: What's new, and why features provide value for upgrading.
...
+
+The Image Assurance feature adds the ability to scan images in container registries at any time, on any infrastructure, including Kubernetes. This is ideal protection for images that don’t go through a pipeline (for example, third-party images), but are published to a registry. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
+
+For more information, see [Scan images in container registries](../image-assurance/scanners/registry-scanner).
+
+#### Security event management
```
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: H3

**Review by ctauchen — COMMENTED** [review]
LGTM, pending heading fixes

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: Fixed

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: Fixed

**Pull request merged**